src/components/PhotoFrame.tsx

```diff
@@ -250,7 +273,8 @@ const PhotoFrame = ({
             h: window.innerHeight,
         };
         if (
-            files[index].metadata.fileType === FILE_TYPE.VIDEO &&
+            (files[index].metadata.fileType === FILE_TYPE.VIDEO ||
+                files[index].metadata.fileType === FILE_TYPE.LIVE_PHOTO) &&
```

Comment:
I think we should add the live photo indicator during loading too?

Reply:
Sure, will do that.